cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorMockTest.java

```diff
@@ -281,7 +283,8 @@ private void init(Params params) {
         ResourceOperatorSupplier supplier = supplierWithMocks();
         ClusterOperatorConfig config = ResourceUtils.dummyClusterOperatorConfig(VERSIONS);
         operator = new KafkaAssemblyOperator(vertx, pfa, new MockCertManager(),
-                new PasswordGenerator(10, "a", "a"), supplier, config);
+                new PasswordGenerator(10, "a", "a"), supplier, config,
+                Mockito.mock(KubernetesRestartEventPublisher.class));
```

Comment:
This is using a Kubernetes Mock. Why does it need to be _mockito mocked_? This should be able to just pass with a regular event publisher.

Reply:
Very good point, I'll check everywhere else I used a mock to see if a) it's needed at all and b) isn't  better handled by the ResourceOperatorSupplier.

Reply:
The tests using the MockKube should have normally `Mock` in the name. So in most cases they should be easy to recognize. For the others, you of course need to use Mockito.